bundle-istio.yaml

```diff
@@ -0,0 +1,16 @@
+bundle: kubernetes
+applications:
+  istio-ingressgateway:
+    charm: cs:~kubeflow-charmers/istio-ingressgateway
+    source: ../bundle-istio/charms/istio-ingressgateway
+    scale: 1
+  istio-pilot:
+    charm: cs:~kubeflow-charmers/istio-pilot
+    source: ../bundle-istio/charms/istio-pilot
+    scale: 1
+  argo-ui:
+    charm: cs:~kubeflow-charmers/argo-ui
+    scale: 1
+relations:
+- [istio-pilot, argo-ui]
```

Comment:
consider being explicit and spell out the relation.  even if there's only 1 today, future Ken will be angry when another compatible interface is added later and this becomes ambiguous.